lib/foreman/renderer.rb

```diff
@@ -26,6 +26,18 @@ def template_logger
       @template_logger ||= Foreman::Logging.logger('templates')
     end
 
+    def host_info(*path)
+      @info ||= @host.info
+      info = @info
+      path.each { |step| info = info.fetch step }
+      info
+    end
+
+    def host_param(*path)
+      path.unshift 'parameters'
+      host_info(*path)
```

Comment:
I think this should be using `@host.params`, not calling .info, that's only for the Puppet ENC output.